compose-builder/Makefile

```diff
@@ -108,9 +113,9 @@ ifeq (ds-onvif-camera, $(filter ds-onvif-camera,$(ARGS)))
 			TOKEN_LIST:=$(TOKEN_LIST),device-onvif-camera
 		endif
 		ifeq ($(KNOWN_SECRETS_LIST),)
-			KNOWN_SECRETS_LIST:=redisdb[device-onvif-camera]
+			KNOWN_SECRETS_LIST:=redisdb[device-onvif-camera],message-bus[device-onvif-camera]
 		else
-			KNOWN_SECRETS_LIST:=$(KNOWN_SECRETS_LIST),redisdb[device-onvif-camera]
+			KNOWN_SECRETS_LIST:=$(KNOWN_SECRETS_LIST),redisdb[device-onvif-camera],message-bus[device-onvif-camera]
 		endif
```

Comment:
@jim-wang-intel , Not for this PR, but thinking we could create a script handle this in on line. Something like:
```
KNOWN_SECRETS_LIST:= ./add_known_secrets($(KNOWN_SECRETS_LIST, device-onvif-camera)
```
Add something similar for TOKEN_LIST. 
These would do the repeated code in one place that check if list is empty..

Reply:
> @jim-wang-intel , Not for this PR, but thinking we could create a script handle this in on line. Something like:
> 
> ```
> KNOWN_SECRETS_LIST:= ./add_known_secrets($(KNOWN_SECRETS_LIST, device-onvif-camera)
> ```
> 
> Add something similar for TOKEN_LIST. These would do the repeated code in one place that check if list is empty..

okay , can create a tech debt issue for this improvement